### PR TITLE
Connected to Firebase

### PR DESCRIPTION
**Pull request by smit-pate-l**

### CODE REVIEW

File: screens/CapturePose.js [hunk 1]
```diff
@@ -1,6 +1,6 @@
 import {useEffect, useRef, useState} from 'react';
 import {StyleSheet, Text, View} from 'react-native';
-import {useNavigation, useIsFocused} from '@react-navigation/native';
+import {useNavigation, useIsFocused, TabRouter} from '@react-navigation/native';
```
Comment (tymonaghan) [inline screens/CapturePose.js:3]: is TabRouter doing anything here?

File: firebase/firestore.js [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+    console.log('error occured in getting user', error);
+  }
+};
+
+export const updateUser = async email => {
+  try {
+    db.collection('users')
+      .doc(email)
+      .set({
+        name: 'Smit Patel',
```
Comment (tymonaghan) [inline firebase/firestore.js:25]: Can we pass in the values from the Login screen instead? Currently, this is setting all new users to have these values.

File: firebase/firestore.js [hunk 3]
```diff
@@ -0,0 +1,43 @@
+import {auth, db} from '../firebase';
+
+export let score = 0;
+
+export const getUser = async () => {
+  try {
+    const users = db.collection('users').doc('hxTAarepPjgInAuZz5VD');
```
Comment (tymonaghan) [inline firebase/firestore.js:7]: Can we have this function take in the user's email or something as a parameter? Currently this is looking up the same record for every user.

File: screens/MyTabs.js [hunk 4]
```diff
@@ -1,17 +1,26 @@
...
 
 const Tab = createBottomTabNavigator();
 
 const HomeScreen = () => {
   const navigation = useNavigation();
 
+  useEffect(async () => {
+    await getUser();
+    console.log(score);
+    await getAllUsers();
```
Comment (tymonaghan) [inline screens/MyTabs.js:21]: Is this something we can assign to a variable and use later for the leaderboard?

File: firebase/firestore.js (hunk 2 above)
Comment (smit-pate-l) [reply firebase/firestore.js:25]: yea we can do that by just replacing the hard coded value with the name

**Review by tymonaghan — APPROVED** [review]

**Pull request merged**